frontstage/views/register.py

```diff
@@ -34,6 +35,8 @@ def validate_enrolment_code(enrolment_code):
 
     if result.status_code == 200 and json.loads(result.text)['active']:
         case_id = json.loads(result.text)['caseId']
+    elif result.status_code != 200:
```

Comment:
Should it be everything other than 200 or should it be more specific? (I know it's probably difficult to say)